services/ui/streamlit_app.py

```
@@ -2039,9 +2119,14 @@ def _load_data(cfg: AppConfig) -> None:
             key="load_zip_unified",
         )
         delete_clicked = c2.button(
-            "Delete site",
+            "Delete…",
             key="delete_dataset_unified",
-            help="Delete Feather/parquet for the active site's building id, then switch to a remaining site. Keeps fault sliders & session maps.",
+            help="Purge Feathers + FDD results for the Active site (building_id). Other sites untouched.",
+        )
+        st.sidebar.checkbox(
+            f"Confirm delete Active site",
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)